examples/resources/rafay_gke_cluster/provider.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_providers {
+    rafay = {
+      "source": "rafay/rafay",
+      "version": "1.1.13"
```

Comment:
@stephan-rafay should it say >= certain version? 
please confirm what this field should be

Reply:
```
terraform {
  required_providers {
    rafay = {
      version = ">= 1.1.15"
      source = "registry.terraform.io/RafaySystems/rafay"
    }
  }
}
```